OpenSim/Simulation/Model/Muscle.h

```diff
@@ -36,8 +36,28 @@
 
 namespace OpenSim {
 
-//=============================================================================
-//=============================================================================
+//==============================================================================
+//                              Muscle Exceptions
+//==============================================================================
+class MuscleCannotEquilibrate : public Exception {
+public:
+    MuscleCannotEquilibrate(const std::string& file,
+                            size_t line,
+                            const std::string& func,
+                            const Object& obj,
+                            const std::string& detail) :
+        Exception(file, line, func, obj, detail) {
+        std::string msg = "Unable to compute equilibrium for this muscle.\n";
+        msg += "Please verify that the initial activation is valid and that ";
+        msg += "the length of the musculotendon actuator doesn't produce a ";
+        msg += "pennation angle of 90 degrees or a negative fiber length.";
+        addMessage(msg);
+    }
+};
+
```

Comment:
> Consider deriving from this exception for each muscle type. The advantage is that when multiple muscles are involved and one of them throws (failure to reach equilibrium), the enclosing try-catch block(s) can catch/do different things (or not) depending on muscle type.

At present, there does not appear to be a need to handle the exception differently depending on muscle type.